src/components/Content/Popup/Popup.js

```diff
@@ -93,7 +95,7 @@ Popup.propTypes = {
 		getBoundingClientRect: PropTypes.func
 	}),
 	/** One of the placements supported by Popper.js relative to target node. */
-	placement: PropTypes.string,
+	placement: PropTypes.oneOfType([ PropTypes.string, PropTypes.bool ]),
```

Comment:
Can PropTypes.bool be replaced with 'false' or PropTypes.exact(false)?

Reply:
react2dts tool cannot properly parse `false` and `PropTypes.exact(false)` resulting in `any` type